packages/elemental-theme/src/custom-elements/ef-slider-marker.less

```
@@ -0,0 +1,12 @@
+:host {
+  width: @slider-step-width;
+  height: @slider-step-width;
+
+  [part='marker'] {
+    background-color: var(--marker-color, @global-text-color);
+  }
+  [part='label'] {
+    top: 15px;
+    transform: translateX(-40%);
```

We should translate according to slider-step-width as well.
```suggestion
    transform: translateX(calc(-50% + @slider-step-width));
```